apps/src/code-studio/assets/SpriteUpload.jsx

```diff
@@ -294,10 +295,10 @@ export default class SpriteUpload extends React.Component {
           )}
 
           <h2 style={styles.spriteUploadStep}>
-            Step 2: Select the sprite to upload
+            Step 2: Select an animation image to upload
           </h2>
           <label>
-            <h3>Select Sprite to Add to Library:</h3>
+            <h3>Choose a file from your computer:</h3>
             <p>Filename cannot contain spaces or capital letters.</p>
```

Comment:
I don't think we want special characters either. For an internal tool, I think that might be obvious enough by implication - since we don't even allow capital letters. Thoughts?